Code/Tools/BundleLauncher/O3DE_SDK_Launcher.cpp

```diff
@@ -46,7 +46,7 @@ int main(int argc, char* argv[])
     shellProcessLaunch.m_processExecutableString = AZStd::move(shellPath.Native());
     shellProcessLaunch.m_commandlineParameters = parameters;
     shellProcessLaunch.m_showWindow = true;
-    shellProcessLaunch.m_workingDirectory = AZStd::string::format("%s", enginePath.c_str());
+    shellProcessLaunch.m_workingDirectory = enginePath.Native();
```

Comment:
This should be `.String()` otherwise it will not compile using Mac clang

Reply:
This does compile for me. I can change it to `.String()` though.

Reply:
I am surprised it compiled and didn't complain about trying to convert from AZStd::basic_fixed_string to AZStd::string.